next-docs/pages/world/account-delegation.mdx

```diff
@@ -1 +1,74 @@
+This function takes three parameters:
+
+- `delegatee`, the address given the privileges.
+  This can be `address(0)` to give permission to all addresses.
+
+- `delegationControlId`, this is usually the `ResourceId` for a `System` that decides whether an attempt to do something by the delegatee on behalf of the delegator is authorized or not.
+  Alternatively, this value can be [`UNLIMITED_DELEGATION`](https://github.com/latticexyz/mud/blob/main/packages/world/src/constants.sol#L23-L25), in which case the delegatee has unlimited authority.
+
+  We have an example of a delegation control `System` in [the `std-delegations` module](https://github.com/latticexyz/mud/tree/main/packages/world-modules/src/modules/std-delegations).
+
+- `initCallData`, call data for a function that is called on the `delegationControlId` to inform it of the new delegation.
+  This [call data](https://docs.soliditylang.org/en/latest/abi-spec.html) includes both the function to call and the parameters with which it is provided.
+
+## Using a delegation
+
+The delegatee can use [`callFrom`](https://github.com/latticexyz/mud/blob/main/packages/world/src/World.sol#L353-L394).
+This function takes three parameters:
+
+- `delegator`, the address on whose behalf the call is happening.
+- `systemId`, the `System` to call.
+- `callData`, the [call data](https://docs.soliditylang.org/en/latest/abi-spec.html) to send the `System`, which includes both the function to call and the parameters with which it is provided.
+
+## Namespace delegation
+
+The owner of a namespace can use [`registerNamespaceDelegation`](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol#L265-L306) for all access to the namespace.
+This functionality is useful, for example, to implement a trusted forwarder for the namespace.
+This is not a security concern, even though it looks like one, because the namespace owner already has full control over any table or `System` in the name space.
+
+## Gotchas and edge cases
+
+There are several edge cases which could trip people up when it comes to delegation.
+
+### Only one delegation per (delegator, delegatee) pair
+
+The table that specifies the `System` to check for delegation ([`world:UserDelegationControl`](https://github.com/latticexyz/mud/blob/main/packages/world/src/codegen/tables/UserDelegationControl.sol)) has two fields in the key: `delegator` and `delegetee`.
+This means that if a user registers one delegation to let a specific `System` authorize requests for one purpose (for example, requests to a specific namespace), and then another delegation to have a different `System` authorize requests for a differet purpose (maybe a different namespace), within the same `World`, the second registration overwrites the first.
```

Comment:
This is wrong, since the table has two keys (`delegator` and `delegatee`), there can be two entries in this case:

`(user, system1) -> delegation1`
`(user, system2) -> delegation2`

Reply:
But the table doesn't have a `System` key, only `delegator` (a.k.a. user) and `delegetee` (https://github.com/latticexyz/mud/blob/main/packages/world/src/codegen/tables/UserDelegationControl.sol#L71-L75). So for a single (`user`, `delegatee`) pair there can only be one delegation.

I'm OK with removing it, but it could trip people up.

Reply:
Oh i see where the confusion comes from - the delegation table stores the delegation control system for each (delegator, delegatee) key. It's up to the delegation control system to decide whether a given call (to any system) is correct, but there can only be one delegation control system per (delegator, delegatee)